### ISSUE TITLE
ALB Ingress controller errors immediately after creating Listener

### ISSUE THREAD

**Issue by namm2** [kubernetes-sigs/aws-load-balancer-controller#1433]
I'm using ALB ingress controller v1.1.8, I've noticed sometimes ALB ingress controller spits out ERROR immediately after creating listener like this:

```
I0916 07:34:45.969358       1 listener.go:110] <namespace>/<name>: creating listener 443
E0916 07:34:46.229358       1 controller.go:217] kubebuilder/controller "msg"="Reconciler error" "error"="failed to reconcile listeners due to failed to reconcile extra certificates on listener arn:aws:elasticloadbalancing:eu-central-1:123456789012:listener/app/dcd7309e-xxxxxxxxx-xxxxxxx-b1d2/xxxxxxxxxxxxxxxx/xxxxxxxxxxxxxxxx: ListenerNotFound: One or more listeners not found\n\tstatus code: 400, request id: 9331c439-39b7-4483-bc14-4ae3f8bfc07d"  "controller"="alb-ingress-controller" "request"={"Namespace":"<namespace>","Name":"<name>"}
```

then some random minutes later it continues

```
I0916 07:46:31.368800       1 rules.go:64] <namespace>/<name>: creating rule 1 on arn:aws:elasticloadbalancing:eu-central-1:123456789012:listener/app/dcd7309e-xxxxxxxxx-xxxxxxx-b1d2/xxxxxxxxxxxxxxxx/xxxxxxxxxxxxxxxx
```

Given the nature of cloud providers, shouldn't it reconcile listener almost immediately?

**Comment by M00nF1sh** [comment]
@namm2 
Hi, it's the expected behavior due to AWS APIs are following a  **eventual consistency** model.
When we created a resource like listener, it may not be available immediately if we have a following up API call to use it immediately. (normally the listeners will be available pretty quick, but such delay do happens sometime and the controller backoff retry to handle it)

**Comment by kishorj** [comment]
@namm2 This should be fixed in v2.0.1 and later releases.
